src/com/twitter/intellij/pants/bsp/FastpassUtils.java

```diff
@@ -98,8 +98,8 @@
 
 
   private static List<String> coursierPart(){
-    return Arrays.asList("launch", "org.scalameta:metals_2.12:latest.stable",
-                         "--main", "scala.meta.internal.pantsbuild.BloopPants",
+    return Arrays.asList("launch", "org.scalameta:fastpass_2.12:latest.stable",
+                         "--main", "scala.meta.internal.fastpass.pantsbuild.BloopPants",
```

Comment:
I'm preparing a PR to add this main class here
```suggestion
                         "--main", "scala.meta.fastpass.Fastpass",
```